public/js/infinite_article_scroll.js

```diff
@@ -36,20 +36,47 @@ $(document).ready(function()
         if(scrollPosition >= documentHeight - 200)
         {
             self.loadingArticles = true;
-            var parameters = (self.offset) ? '?offset=' + self.offset + '&limit=' + self.limit : '?';
-            if(typeof infiniteScrollSection !== 'undefined') {
-                if(parameters.length) {
-                    parameters += '&';
-                }
-                parameters += 'section=' + infiniteScrollSection;
+            var parameters = [];  
+
+            //Offset
+            if(!self.offset && typeof infiniteScrollOffset !== 'undefined'){
+                self.offset = infiniteScrollOffset;
             }
-            else if(typeof infiniteScrollTopic !== 'undefined') {
-                if(parameters.length) {
-                    parameters += '&';
-                }
-                parameters += 'topic=' + inifiniteScrollTopic;
+            if(self.offset){
+                parameters.push({offset: self.offset });
+            }
+
+            //Limit
+            if(!self.limit && typeof infiniteScrollLimit !== 'undefined'){
+                self.limit = infiniteScrollLimit;
+            }
+            if(self.limit){
+                parameters.push({limit: self.limit });
             }
-            $.getJSON('/api/content/get_articles' + parameters, function(result)
+            
+            //Section or Topic
+            if(typeof infiniteScrollSection !== 'undefined'){
+                parameters.push({section: infiniteScrollSection });
+            }
+            else if(typeof infiniteScrollTopic !== 'undefined'){
+                parameters.push({topic: infiniteScrollTopic });
+            }
+
+            var query;
```

Comment:
Since this runs off of jQuery, I suggest we use the [param() function](http://api.jquery.com/jquery.param/) to build the string.